lib/suspenders/app_builder.rb

```diff
@@ -113,10 +113,6 @@ def set_up_hound
       copy_file "hound.yml", ".hound.yml"
     end
 
-    def configure_newrelic
```

Comment:
I think that makes sense. I'll look around for an application with this setup to use an example.